bin/_mb.bat

```diff
@@ -29,7 +29,7 @@ if "%PYBLISH_NUKE%"=="" goto :missing
 if "%PYBLISH_LITE%"=="" goto :missing
 if "%PYBLISH_QML%"=="" goto :missing
 if "%MINDBENDER_CORE%"=="" goto :missing
-:: if "%PYBLISH_USERNAME%"=="" goto :missing
+if "%PYBLISH_USERNAME%"=="" goto :missing
```

Comment:
These are the kinds of things we should first make an issue thread about, to discuss, and then implement. Once implemented, it's important to highlight these additions in the PR description, as detailed as necessary. With examples and motivation if necessary.